docs/pre-work/README.md

```diff
@@ -150,6 +150,8 @@ Running the lab notebooks remotely using [Google Colab](https://colab.research.g
 - [Colab Prerequisites](#colab-prerequisites)
 - [Serving the Granite AI Models for Colab](#serving-the-granite-ai-models-for-colab)
 
+!!! note "Slow network tip" If your network connection is slow, consider using a different Colab runtime. From the navigation bar, select `Runtime->Change runtime type`, then select either GPU- or TPU-based hardware acceleration.
```

Comment:
Is this really a slow _network_ issue? Seem more like slow execution.

Reply:
True, I think I need to rephrase this. It's not necessarily a direct fix for the slow network but a way to compensate for it. Change incoming